tests/test_pep.py

```diff
@@ -12,12 +12,30 @@
     ('patch',    '1.2.3', '1.2.4'),
 
     # pre-base version
-    ('premajor', '1.2.3', '2.0.0rc1'),
-    ('preminor', '1.2.3', '1.3.0rc1'),
-    ('prepatch', '1.2.3', '1.2.4rc1'),
+    ('premajor', '1.2.3', '2.0.0alpha1'),
+    ('preminor', '1.2.3', '1.3.0alpha1'),
+    ('prepatch', '1.2.3', '1.2.4alpha1'),
+
+    # add pep specific parts
+    ('pre',      '1.2.3', '1.2.3alpha1'), # pre will now append the first value in ('alpha', 'beta', 'rc')
+    ('alpha',    '1.2.3', '1.2.3alpha1'),
+    ('beta',     '1.2.3', '1.2.3beta1'),
+    ('rc',       '1.2.3', '1.2.3rc1'),
+
+    # update alpha -> beta -> rc
+    ('beta',     '1.2.3alpha1', '1.2.3beta1'),
+    ('rc',       '1.2.3beta1',  '1.2.3rc1'),
+
+    # bump alpha, beta, rc
+    ('alpha',    '1.2.3alpha1', '1.2.3alpha2'),
+    ('pre',      '1.2.3alpha1', '1.2.3alpha2'),
+    ('beta',     '1.2.3beta1',  '1.2.3beta2'),
+    ('pre',      '1.2.3beta1',  '1.2.3beta2'),
+    ('rc',       '1.2.3rc1',    '1.2.3rc2'),
+    ('pre',      '1.2.3rc1',    '1.2.3rc2'),
 
     # add special part
-    ('pre',      '1.2.3', '1.2.3rc1'),
+    #('pre',      '1.2.3', '1.2.3rc1'), # disabled because of changes in bump_pre implementation
```

Comment:
Hmm, let's not make breaking changes and use `rc` by default. Also, I'd expect it to be `rc` just because it's more often used. As a side note, I think there are much fewer projects where pre-release stage is separated by 2, `alpha` and `beta`.

Reply:
Yes, you're right. A default for `rc` would be more natural and more import would not introduce a breaking change.

I agree, I rarely saw projects with an `alpha` build except of games and bigger software packages. Nevertheless, `alpha` builds are supported for the PEP format and therefore I decided, it would be best to implement it too.